Bottom: ac5a5037171dc55c3f05160c852146ea3b1dd131
Top:    a986affdbf41c0e1a5ba4d3673019b6be31321f3
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-31 19:06:44 +0800

Refresh of autofs-5.0.7-teach-dumpmaps-to-output-simple-key-value-pairs.patch

---

diff --git a/lib/master.c b/lib/master.c
index 92bcd56..0c55f21 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1333,7 +1333,8 @@ static int match_type(const char *source, const char *type)
 {
 	if (!strcmp(source, type))
 		return 1;
-	if (!strcmp(type, "file") && !strcmp(source, "files"))
+	/* Source file and files are synonymous */
+	if (!strncmp(source, type, 4) && strlen(source) <= 5)
 		return 1;
 	return 0;
 }
